bitnami/thanos/Chart.yaml

```
@@ -28,4 +28,4 @@ name: thanos
 sources:
   - https://github.com/bitnami/bitnami-docker-thanos
   - https://thanos.io
-version: 9.0.9
+version: 9.0.10
```

Missing new line at end of file:
```suggestion
version: 9.0.10

```